cmd/agent/main.go

```diff
@@ -29,7 +29,8 @@ func init() {
 }
 
 func main() {
-	// If Windows is trying to run us as a service, go through that
+
+	// If Windows is trying to runFlow us as a service, go through that
```

Comment:
runFlow us? :)

Reply:
Should we leave a TODO to allow Flow to work as a Windows service?

Reply:
Silly goland rename and me not double checking. Added a todo but likely not supported til flow graduates to full support.